canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.scss

```diff
@@ -202,12 +212,23 @@
 			padding-left: 3px;
 		}
 		.properties-vt-row-selected {
+			cursor: pointer;
 			border-left: 4px solid $button-primary;
 			padding-left: 0;
 			&:not(.properties-vt-row-disabled):hover {
 				border-left: 4px solid $button-primary;
 			}
 		}
+		.properties-vt-row-doubleClick{
+			border-left: 4px solid $button-primary;
+			cursor: pointer;
+			&:focus,
+			&:active {
+				color: $button-primary;
+				border: 1px solid $button-primary;
+				background-color: $background-active;
+			}
+		}
 	}
```

Comment:
@matthoward366 I tried to add `:focus` and` :active` here. `.properties-vt-row-doubleClick` class is being detected, but the style of :focus and :active not working.